config.go

```diff
@@ -285,6 +308,12 @@ func pluginCmd(path string) *exec.Cmd {
 		}
 	}
 
+	// No plugin binary found, so try to use an internal plugin.
+	if strings.Contains(path, command.TFSPACE) {
+		parts := strings.Split(path, command.TFSPACE)
+		return exec.Command(parts[0], parts[1:]...)
+	}
```

Comment:
This involves a slightly more complicated refactor so I'm inclined to defer this. There is currently logging when internal plugins are used but to differentiate between the various types of disk-based plugins will require some changes to the discovery logic.